src/header/header.scss

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../user-icon/index';
+@import '../icon/index';
+
+
+@mixin iui-header {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: $iui-baseline * 5;
+  transition: height $iui-speed-fast ease-out;
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-bottom: $iui-xxs solid t(iui-color-background-5);
+    color: t(iui-text-color);
+  }
+
+  > .iui-left,
+  > .iui-center,
+  > .iui-right {
+    display: flex;
+    align-items: center;
+    height: 100%;
+    width: 33%;
+  }
+
+  > .iui-left {
+    &:nth-last-child(2) {
+      width: unset;
+      overflow: hidden;
+    }
+    
+    > .iui-application,
+    > .iui-header-button {
+      overflow: hidden;
+      transition: height $iui-speed-fast ease-out;
+
+      > .iui-label {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        transition: font-size $iui-speed-fast ease-out;
+
+        > * {
+          width: 100%;
+          overflow: hidden;
+          text-overflow: ellipsis;
+          text-align: left;
+        }
+      }
+
+      > .iui-brand-logo,
+      > .iui-thumbnail-area {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100%;
+
+        > .iui-icon,
+        > .iui-thumbnail {
+          width: $iui-icons-large;
+          height: $iui-icons-large;
+          transition: width $iui-speed-fast ease-out, height $iui-speed-fast ease-out;
+        }
+      }
+    }
+
+    > .iui-application {
+      padding: 0;
+
+      > .iui-brand-logo {
+        width: $iui-l * 3 - 1;
+        flex-shrink: 0;
+      }
+
+      > .iui-label {
+        margin-right: 12px;
+        transition: margin-right $iui-speed-fast ease-out, font-size $iui-speed-fast ease-out;
+      }
+    }
+
+    > .iui-header-button {
+      > .iui-thumbnail-area {
+        width: $iui-l;
+        margin-right: $iui-sm;
+
+        > .iui-thumbnail {
+          border-radius: $iui-border-radius;
+          background-position: center;
+          background-size: cover;
+          @include themed {
+            background-color: t(iui-color-background-3);
+          }
+        }
+      }
+
+      > .iui-label {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        transition: padding-top $iui-speed-fast ease-out;
+
+        > .iui-id {
```

Comment:
What about having here muted text color? Ofc, for active button we need different color.
![image](https://user-images.githubusercontent.com/36186912/116055323-42697200-a685-11eb-98d7-943a36b7584d.png)

Reply:
The muted text variable includes color & opacity.  I've applied the opacity only so that it doesn't override the active blue color.
![Screen Shot 2021-04-27 at 4 48 09 PM](https://user-images.githubusercontent.com/849817/116310797-51245780-a75f-11eb-9d12-69c22cc26cbc.png)